crates/forge/assets/vyper/workflowTemplate.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  FOUNDRY_PROFILE: ci
+
+jobs:
+  check:
+    name: Foundry project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+
+      - name: Show Forge version
+        run: |
+          forge --version
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+          architecture: x64
```

Comment:
is this necessary?

Reply:
the `python-version` field is: https://github.com/foundry-rs/foundry/pull/11383/commits/81fc919b2794da4caf4efb518d01b4f3f68cf82a

https://github.com/zerosnacks/vyper-template-11383/actions/runs/17131913992/job/48598167191#step:5:13